cprover_bindings/src/goto_program/stmt.rs

```diff
@@ -223,7 +215,12 @@ impl Stmt {
         Stmt::block(
             vec![
                 // Assert our expected true expression.
-                Stmt::assert(expect_true.clone(), &assert_msg, loc.clone()),
+                Stmt::assert_statement(
+                    expect_true.clone(),
+                    "sanity_check",
```

Comment:
Is this meant to be hard-coded?

Reply:
I couldn't come up with a better solution tbh. Open to suggestions about how to address `property_class` values in cprover_bindings. Maybe I can use the PropertyClass Enum values from codegen here?

Reply:
No worries... I think we discussed that before, didn't we? Sorry

Reply:
We agreed to separate the `enum` from `cprover_bindings` and place it in codegen, but beyond that i dont think we discussed about calling the `enum` from `codegen` again in `cprover_bindings`. I can do that but i feel like it would remove the seperation of logic, which is why i just hardcoded the string. 

open to any suggestions regarding this.